bin/run_ingestion.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+#email_recipient=***********
+#eva_dir=**************
+# Grab the variables from a config bash script
+source ~/.covid19dp_processing 2> /dev/null
```

Comment:
Why redirect errors to /dev/null? Is it to suppress any extraneous message that might come from the script?

Reply:
You're right it's unnecessary. I'll remove it.